bin/test.py

```diff
@@ -1,17 +1,16 @@
 #!/usr/bin/env python
 
-import unittest
-import sys
 import os
+import subprocess
+import sys
 
 
 def main():
-    loader = unittest.TestLoader()
-    tests = loader.discover(os.path.join(sys.path[0], '../test'))
-
-    runner = unittest.TextTestRunner()
-    runner.run(tests)
+    return subprocess.call(
```

Comment:
I have a question here, and this is more for my own benefit as a coder, but why is it better to run the tests in a subprocess rather than in the parent?